zschema/tests.py

```diff
@@ -127,8 +127,7 @@ def assertBigQuerySchemaEqual(self, a, b):
         BigQuery schemas consist of lists whose order doesn't matter, dicts,
         and privimites.
         """
-        if type(a) is not type(b):
-            return False
+        self.assertEquals(type(a), type(b))
```

Comment:
I'm still also worrisome that this logic is un-pythonic and apt to break, for example in different cases of `basestring`.